dp/generate_climos.py

```diff
@@ -61,34 +60,48 @@ def input_check(filepath, climo):
         return None, []
 
     periods = input_file.climo_periods.keys() & climo
-    logger.info("climo_periods: {}".format(periods))
+    outputer("climo_periods: {}".format(periods))
     if len(periods) == 0:
-        logger.critical(f"{input_file.filepath()} contains no standard climatological periods")
+        logger.critical(
+            f"{input_file.filepath()} contains no standard climatological periods"
+        )
         return None, []
 
     return input_file, periods
 
 
-def dry_run_handler(filepath, climo):
-    '''
+def dry_run_handler(filepath, climo, outpath=None):
```

Comment:
Above you defaulted this value to `logger.info`, I think we could do that here as well.